Configuration/PyReleaseValidation/python/relval_gpu.py

```diff
@@ -71,3 +71,5 @@
 workflows[141.008506] = ['Run3-2023_JetMET2023B_RecoPixelOnlyTripletsGPU',['RunJetMET2023B','HLTDR3_2023','RECODR3_reHLT_Patatrack_PixelOnlyTripletsGPU','HARVESTRUN3_pixelTrackingOnly']]
 workflows[141.008512] = ['Run3-2023_JetMET2023B_RecoECALOnlyGPU',['RunJetMET2023B','HLTDR3_2023','RECODR3_reHLT_ECALOnlyGPU','HARVESTRUN3_ECALOnly']]
 workflows[141.008522] = ['Run3-2023_JetMET2023B_RecoHCALOnlyGPU',['RunJetMET2023B','HLTDR3_2023','RECODR3_reHLT_HCALOnlyGPU','HARVESTRUN3_HCALOnly']]
+
+workflows[160.502] = ['',['HydjetQ_MinBias_5362GeV_2023_ppReco','DIGIHI2023PPRECO','RAWPRIMESIMHI18','RECOHI2023PPRECOMB_PatatrackGPU','MINIHI2023PROD']]
```

Comment:
does this include the same changes as the `.502` workflow used for pp ?

Reply:
Yes (see [here](https://github.com/cms-sw/cmssw/blob/4fac7bb211bfbbd03d35543221c44a7c3b1a8e1e/Configuration/PyReleaseValidation/python/relval_steps.py#L3010)).

Reply:
Can you add a one-line comment similar to the ones above ?